pom.xml

```diff
@@ -3,7 +3,9 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.509.1</version><!-- which version of Jenkins is this plugin built against? -->
+    <version>1.580.2</version><!-- which version of Jenkins is this plugin built against? (Current LTS as at 14 Jan 2015)-->
```

Comment:
This value defines the minimal jenkins core on what this plugin can be run. Don't update it without any critical reason.
Such update cuts plugin usage for thousands installations.